### PR TITLE
Refactor HDP to HBP

### PR DESCRIPTION
**Pull request by grandizzy**
Recalculate HBP on removeQuoteToken and purchaseBid calls
Fix make lint command to use path to prettier package

### CODE REVIEW

File: src/ERC20Pool.sol [hunk 1]
```diff
@@ -590,7 +604,9 @@ contract ERC20Pool is IPool, Clone {
     function getLPTokenBalance(address _owner, uint256 _price)
         external
         view
-        returns (uint256 lpTokens) // RAY
+        returns (
```
Comment (EdNoepel) [inline src/ERC20Pool.sol]: What's going on with the linting here?  This is < 80 chars, and we're wrapping at 100.

File: src/ERC20Pool.sol [hunk 2]
```diff
@@ -233,6 +233,11 @@ contract ERC20Pool is IPool, Clone {
             lup = newLup;
         }
 
+        // update HBP if removed from current, if no deposit nor debt in current HBP and if LUP not 0
+        if (_price == hbp && bucket.onDeposit == 0 && bucket.debt == 0 && lup != 0) {
+            hbp = getHbp();
+        }
```
Comment (EdNoepel) [inline src/ERC20Pool.sol:239]: looks good

File: src/ERC20Pool.sol [hunk 3]
```diff
@@ -438,6 +444,11 @@ contract ERC20Pool is IPool, Clone {
...
```
Comment (EdNoepel) [inline src/ERC20Pool.sol:450]: looks good

**Review by EdNoepel — COMMENTED** [review]
Appreciate renaming for consistency with sim.  HBP now updated by pool contract; Buckets library doesn't have a say in it.  One comment regarding a linting issue.

File: src/ERC20Pool.sol (hunk 1 above)
Comment (grandizzy) [reply src/ERC20Pool.sol]: @EdNoepel not sure exactly why linter like this when comment like`//RAY` I moved types in NatSpec and it looks nicer now

**Review by EdNoepel — APPROVED** [review]
Nice

**Pull request merged**